interfaces/builtin/pulseaudio.go

```diff
@@ -64,7 +64,7 @@ const pulseaudioConnectedPlugAppArmorDesktop = `
 /etc/pulse/** r,
 owner @{HOME}/.pulse-cookie rk,
 owner @{HOME}/.config/pulse/cookie rk,
-owner /{,var/}run/user/*/pulse/ rwk,
+owner /{,var/}run/user/*/pulse/ rk,
```

Comment:
This probably only needs `r` permission.

Reply:
You are right. I'll test it (in both `pulseaudio.go` and `audio-playback.go`) and, if everything works as expected, will modify the patch.

Reply:
Tested with Firefox and Telegram in a classic ubuntu, and with firefox and the gnome-control-panel in core desktop. In all cases it works fine.